schedule/hpc/installation_server.yaml

```diff
@@ -0,0 +1,53 @@
+---
+name: hpc_installation_server
+description:    >
+     Maintainer: schlad
+     Installation scenario with HPC system role hpc-server.
+vars:
+  DESKTOP: textmode
+  INSTALLONLY: 1
+  PATTERNS: base,minimal
+  SLE_PRODUCT: hpc
+  HDDSIZEGB: 30
+conditional_schedule:
+  bootloader:
```

Comment:
@schlad to avoid this there is installation/bootloader_start

Reply:
oh thx, I did not know! will try it out.

Reply:
hmmm but...
"# Summary: Temporary solution to combine all the bootloader modules in one and
# schedule them depending on the environment variables.
# The solution is implemented to use in declarative scheduling which does not
# allow to use complex conditions.

the code in that .pm calls anyway different bootloaders openqa modules.

So perhaps there are mode complex conditions, but for this hpc/installation_server.yaml the condition for bootloader is so super simple, that I honestly don't see any advantage of using that 'wrapper'. Or am I missing something?

Reply:
jfyi that it exists I got same hint from @rwx788